src/index.js

```diff
@@ -1,152 +1,26 @@
+import listbooks from './modules/localStorage.js';
+import listNavMenu from './modules/listMenuComponent.js';
+import addMenu from './modules/addMenuComponent.js';
+import contactNavMenu from './modules/contactMenuComponent.js';
+import luxon from './modules/luxun.js';
+
+const { DateTime } = luxon;
+const now = DateTime.now();
+
+console.log(now.c);
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you remove all the `console.log()` for **[JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)** best practices. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.

Reply:
**Thanks, @tresorsawasawa,** 
I forgot to remove that console statement.
 **I will make sure to remove it.**